modules/network-observability-flowcollector-about.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-flowcollector-about{context}"]
+
+= About the FlowCollector resource 
+The  `FlowCollector` resource is deployed at the time of installation. You can edit yaml directly in the {product-title} web console. In the web console, navigate to *Operators* -> *Installed Operators*. Then under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. Next, select *cluster* then select the *YAML* tab. There, you can modify the `FlowCollector` resource to configure the Network Observability operator. Some of the notable configurations are documented in this documentation. 
+
+The following example shows a sample `FlowCollector` resource for {product-title} Network Observability operator:
+[id="network-observability-flowcollector-configuring-about-sample_{context}"]
+.Sample `FlowCollector` resource
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF                                <1>
+    ebpf:
+      sampling: 50                            <2>
+      logLevel: info
+      privileged: false
+      resources:
+        requests:
+          memory: 50Mi
+          cpu: 100m
+        limits:
+          memory: 800Mi
+      kafkaBatchSize: 10485760
+  processor:
+    logLevel: info
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+      limits:
+        memory: 800Mi
+  loki:               <3>
+    url: 'http://loki.netobserv.svc:3100/'
+    statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
+    authtoken: HOST
+    tls:
+      enable: false
```

Comment:
This Loki section wouldn't work; when Loki is deployed with the operator, here's how it should be configured:

```suggestion
  loki:               <3>
    url: 'https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network/'
    statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100'
    tenantID: network
    authToken: HOST
    tls:
      enable: true
      caCert:
        type: configmap
        name: loki-ca-bundle
        certFile: service-ca.crt
```

Reply:
Addressed

Reply:
Hi @skrthomas 
There are two issues remaining in this Loki config: the `url` is still not correct (it should be `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network/`), and `authToken` is misspelled `authtoken` (missing capital T)
With these to changes we should be good with the loki config

Reply:
Ah, I see. Addressed these two lingering issues.